get-started/the-stack.md

```diff
@@ -10,92 +11,86 @@ products:
+{{es}} is the heart of the Elastic Stack, functioning as the central place to store and search your data. It stores data as **JSON documents**, which are structured data objects. These documents are organized into **indices**, which you can think of as collections of similar documents.
 
-#### {{ls}} [stack-components-logstash]
+Elasticsearch is built to be a resilient and scalable distributed system. It runs as a **cluster** of one or more servers, called **nodes**. When you add data to an index, it's divided into pieces called **shards**, which are spread across the various nodes in the cluster. This architecture allows Elasticsearch to handle large volumes of data and ensures that your data remains available even if a node fails.
 
-{{ls}} is a data collection engine with real-time pipelining capabilities. It can dynamically unify data from disparate sources and normalize the data into destinations of your choice. {{ls}} supports a broad array of input, filter, and output plugins, with many native codecs further simplifying the ingestion process. [Learn more about {{ls}}](logstash://reference/index.md).
+Learn more in [The {{es}} data store](/manage-data/data-store.md)
 
+### Visualize and query your data [kibana-navigation-search]
 
-### Store [_store]
+While {{es}} stores your data, **Kibana** is the user interface where you can explore, visualize, and manage it. It provides a window into your data, allowing you to quickly gain insights and understand trends.
 
-#### {{es}} [stack-components-elasticsearch]
+With Kibana, you can:
 
-{{es}} is the distributed search and analytics engine at the heart of the {{stack}}. It provides near real-time search and analytics for all types of data. Whether you have structured or unstructured text, numerical data, or geospatial data, {{es}} can efficiently store and index it in a way that supports fast searches. {{es}} provides a REST API that enables you to store data in {{es}} and retrieve it. The REST API also provides access to {{es}}'s search and analytics capabilities. [Learn more about {{es}}](/get-started/index.md).
+* Use **Discover** to interactively search and filter your raw data.  
+* Build custom **visualizations** like charts, graphs, and metrics with tools like **Lens**, which offers a drag-and-drop experience.  
+* Assemble your visualizations into interactive **dashboards** to get a comprehensive overview of your information.  
+* Analyze geospatial data using the powerful **Maps** application.
 
+At the same time, Kibana works as the user interface of all Elastic solutions, like Elastic Security and Elastic Observability, providing ways of configuring Elastic to suit your needs and offering interactive guidance.
 
-### Consume [_consume]
+A **query** is a question you ask about your data, and Elastic provides several powerful languages to do so. You can query data directly through the API or through the user interface in Kibana.
```

Comment:
this is missing querydsl, the most common way people query data currently

Reply:
Added and reordered the list according to our docs on query langs